README.md

```diff
@@ -49,12 +49,17 @@ The JSON InSpec results file may also be loaded into a __full heimdall server__,
 To report a bug or feature request, please open an [issue](https://github.com/mitre/canonical-ubuntu-18.04-lts-stig-baseline/issues/new).
 
 ## Authors
-- Aaron Lippold
-- The MITRE InSpec Team
+- [Aaron Lippold](https://github.com/aaronlippold)
```

Comment:
Please update the authors to include yourself, and feel free to move me to the special thanks or reviewer section.